src/homework/eighth/forkexample/Model.java

```diff
@@ -0,0 +1,31 @@
+package homework.eighth.forkexample;
+
+import java.util.concurrent.RecursiveTask;
+
+public class Model extends RecursiveTask<Long> {
+    private long from;
+    private long to;
+
+    public Model(long from, long to) {
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    protected Long compute() {
+        if ((to - from) <= Service.getCountOfOperation()  / Service.getNumberOfThreads()) {
```

Comment:
Please explain how does it work?

Reply:
In overriding of method compute() we determing if there is need for 
 using fork, and for using many threads for performing the task (in "if" part). In second part of method we discribe how to conduct task by using fork.